packages/components-js/tests/e2e/puppeteer/specs/select-wrapper-filter.e2e.ts

```diff
@@ -128,7 +128,8 @@ describe('focus state', () => {
   });
 });
 
-describe('hover state', () => {
+// puppeteer ignores @media(hover: hover) styles, but playwright can handle it
+xdescribe('hover state', () => {
```

Comment:
Isn't is also disabling the next test case "should render dropdown list if filter input is clicked and remove via outside click", which is unrelated to hover state?